gallery/src/pageConfig.ts

```diff
@@ -140,141 +140,151 @@ import NxStatefulCollapsibleRadioSelectPage
-    'nx-counter': NxCounterPage,
-    'nx-divider': NxDividerPage,
-    'nx-fieldset': NxFieldsetStylePage,
-    'nx-form-group': NxFormGroupStylePage,
-    'nx-form-select': NxFormSelectPage,
-    'nx-h*': NxHPage,
-    'nx-global-header': NxGlobalHeaderPage,
-    'nx-grid': NxGridPage,
-    'nx-icon': NxIconPage,
-    'nx-list': NxListPage,
-    'nx-p': NxPPage,
-    'nx-page-title': NxPageTitlePage,
-    'nx-pre': NxPrePage,
-    'nx-read-only': NxReadOnlyPage,
-    'nx-system-notice': NxSystemNoticePage,
-    'nx-table': NxTableStylePage,
-    'nx-table-container': NxTableContainerPage,
-    'nx-threat-number': NxThreatNumberPage,
-    'nx-tile': NxTilePage
+  'Layout': {
+    'Form Layout Examples': { content: NxFormLayoutPage, type: 'layout' },
+    'Page Layout Examples': { content: PageLayoutPage, type: 'layout' },
+    'Read-Only Grid Tile Layout': { content: NxReadOnlyGridTilePage, type: 'layout' },
+    'Card': { content: NxCardPage, type: 'html' },
+    'Divider': { content: NxDividerPage, type: 'html' },
+    'Global Header': { content: NxGlobalHeaderPage, type: 'html' },
+    'Global Sidebar': { content: NxGlobalSidebarPage, type: 'react' },
+    'Stateful Global Sidebar': { content: NxStatefulGlobalSidebarPage, type: 'react' },
+    'Global Sidebar Footer': { content: NxGlobalSidebarFooterPage, type: 'react' },
+    'Grid': { content: NxGridPage, type: 'html' },
+    'Modal': { content: NxModalPage, type: 'react' },
+    'Nexus Page Header': { content: NxNexusPageHeaderPage, type: 'react' },
+    'Page Header': { content: NxPageHeaderPage, type: 'react' },
+    'Page Title': { content: NxPageTitlePage, type: 'html' },
+    'read-only': { content: NxReadOnlyPage, type: 'html' },
+    'Tabs': { content: NxTabsPage, type: 'react' },
+    'Stateful Tabs': { content: NxStatefulTabsPage, type: 'react' },
+    'Tile': { content: NxTilePage, type: 'html' },
+    'Vulnerability Details': { content: NxVulnerabilityDetailsPage, type: 'react' }
   },
-  'Styles - Mixins & Helpers': {
-    'Color Palettes': ColorPalettePage,
-    'Custom app font size': NxFontSizePage,
-    'Ellipsis Truncation': NxTruncatePage,
-    'nx-clickable': NxClickablePage,
-    'nx-container-helpers': NxContainerHelpersPage,
-    'nx-scrollable': NxScrollablePage,
-    'nx-viewport-sized': NxViewportSizedPage,
-    'Selectable Colors': SelectableColorsPage,
-    'Style Variables': CssVariablesPage
+  'Typography and Icons': {
+    'Font Awesome Icon': { content: NxFontAwesomeIconPage, type: 'react' },
+    'Blockquote': { content: NxBlockquotePage, type: 'html' },
+    'Code': { content: NxCodePage, type: 'html' },
+    'H*': { content: NxHPage, type: 'html' },
+    'P': { content: NxPPage, type: 'html' },
+    'Pre': { content: NxPrePage, type: 'html' }
   },
-  'Layout Examples': {
-    'Form Layout Styles': NxFormLayoutPage,
-    'Page Layout': PageLayoutPage,
-    'Read-Only Grid Tile Layout': NxReadOnlyGridTilePage
+  'HTML Variants': {
```

Comment:
Is this category temporary until RSC-842 is worked out?

Reply:
> Is this category temporary until RSC-842 is worked out?

Yes exactly.